ush/test/README.md

```diff
@@ -33,20 +29,22 @@ This is a prototype for testing BUFR to IODA conversion and is still evolving.
    ```
 
 ## Elements should be in the working directory from SPOC
-- Required input files:
+Creat a working directory (e.g. work_dir)
+
+- Required input files in ./work_dir:
   
-   - bufr_satwnd_amv_goes.py
+   - bufr_satwnd_amv_goes.py (copied /spoc/dump/mapping)
```

Comment:
why copying things?

Reply:
This is a stand-alone test.  So, the uses need to gather (copy) the required input files (bufr, mapping, python script, configuration files)  in their working directory along with the test script (encodeBufr.sh) to run the test.